src/output.rs

```diff
@@ -0,0 +1,66 @@
+use crate::card::Card;
+use crate::hand::Hand;
+use crate::hand_score::HandScore;
+use crate::hand_stats::HandStats;
+use poker_stats::aggregate_score::sample_aggregate_scores;
+use rand::rngs::ThreadRng;
+
+pub fn compute<const CARD_NUMBER: usize>(mut rng: ThreadRng) {
```

Comment:
Make naming more clear.  (`compute` is too generic)
```suggestion
pub fn draw_and_display_hand<const CARD_NUMBER: usize>(mut rng: ThreadRng) {
```